### PR TITLE
Added yml file

### PR DESCRIPTION
**Pull request by aec1140**

### CODE REVIEW

File: people/2017/spring/swaggy.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#hfoss
+name: Alex Cook
+blog: https://aec1140.wordpress.com/
+feed: https://aec1140.wordpress.com/
+email: aec1140@gmail.com
+major: Game Design and Development
+irc: swaggy
+rit_dce: None
```
Comment (jflory7) [inline people/2017/spring/swaggy.yml]: For `rit_dce`, this indicates your RIT computer account / username, e.g. aec1140. You'll want to specify that here in this field.

File: people/2017/spring/swaggy.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+feed: https://aec1140.wordpress.com/
+email: aec1140@gmail.com
+major: Game Design and Development
+irc: swaggy
+rit_dce: None
+forges:
+  - https://github.com/aec1140
+bio:  4th year Game Design and Development student looking for Government Simulation jobs.
+hw:
+firstflight: https://aec1140.wordpress.com/2017/01/25/first-blog-post/
```
Comment (jflory7) [inline people/2017/spring/swaggy.yml]: As per YAML syntax, you'll want to do an indention here (two spaces) so it's underneath the `hw` section.

File: people/2017/spring/swaggy.yml (hunk 1 above)
Comment (ritjoe) [inline people/2017/spring/swaggy.yml]: I'm OK with None for the dce here.

**Comment by ritjoe** [comment]
Those are carriage returns. To read more about the various line-ending conventions see, for instance https://en.wikipedia.org/wiki/Newline

The biggest issues with this submission is the whitespace, as noted by @jflory7 and the filename extension: This needs to be swaggy.yaml not swaggy.yml

File: people/2017/spring/swaggy.yml (hunk 1 above)
Comment (jflory7) [inline people/2017/spring/swaggy.yml]: @ritjoe Ah, okay. I just remembered this from past semesters and wasn't sure if it was still relevant. Could possibly just remove the field for future semesters, but that's probably more meta than the scope of this pull request.

**Comment by aec1140** [comment]
I think everything is all set now

File: people/2017/spring/swaggy.yaml [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
-forges:
-  - https://github.com/aec1140
-bio:  4th year Game Design and Development student looking for Government Simulation jobs.
-hw:
-firstflight: https://aec1140.wordpress.com/2017/01/25/first-blog-post/
\ No newline at end of file
+#hfoss
+name: Alex Cook
+blog: https://aec1140.wordpress.com/
+feed: https://aec1140.wordpress.com/
```
Comment (jflory7) [inline people/2017/spring/swaggy.yaml]: One more thing, didn't catch this until now! The feed of your blog is `https://aec1140.wordpress.com/feed/`, which is an RSS feed. The [HFOSS Planet](https://people.rit.edu/djaigm/planet/hfoss/) needs an RSS feed to put your blog posts there. Although, if you are using categories for your blog, you may want to use an appropriate category for HFOSS posts so _all_ of your blog posts don't wind up there (e.g. `https://aec1140.wordpress.com/category/foss/feed/`). Sorry for not catching this earlier.

Comment (jflory7) [inline people/2017/spring/swaggy.yaml]: One more thing, didn't catch this until now! The feed of your blog is `https://aec1140.wordpress.com/feed/`, which is an RSS feed. The [HFOSS Planet](https://people.rit.edu/djaigm/planet/hfoss/) needs an RSS feed to put your blog posts there. Although, if you are using categories for your blog, you may want to use an appropriate category for HFOSS posts so _all_ of your blog posts don't wind up there (e.g. `https://aec1140.wordpress.com/category/foss/feed/`). Sorry for not catching this earlier.

**Pull request merged**